config/locales/en.yml

```diff
@@ -36,6 +36,10 @@ en:
   time:
     formats:
       default: "%d-%^b-%Y %I:%M %p"
+  errors:
+    messages:
+      password:
+        password_strength: "is too weak. Please avoid common words and reptitive / sequential characters."
```

Comment:
Copy suggestion: `This password is too easy to guess. Change your password to something stronger, like "<passphrase>"`

Reply:
Updated with that.